src/subHeader/story.tsx

```diff
@@ -1,17 +1,12 @@
 import React from 'react'
-import { withKnobs } from '@storybook/addon-knobs'
 import { storiesOf } from '@storybook/react'
 
 import { BaseSection as Section } from '../layout/section/baseSection'
 import { SubHeader } from '../subHeader'
-import spec from './specifications/subHeader.md'
 
-const stories = storiesOf('Widgets|SubHeader', module)
-stories.addDecorator(withKnobs)
-const specTemplateFn = () => <SubHeader>SubHeader content</SubHeader>
-stories.add('Specifications', specTemplateFn, { readme: spec })
+const stories = storiesOf('Text|', module)
```

Comment:
Why not keep Text|SubHeader like other stories ?

Reply:
'cos "Default" can be declared direct as story, no need to create another level of hierarchy